lib/shoryuken/cli/install.rb

```diff
@@ -0,0 +1,31 @@
+require 'thor/group'
+
+module Shoryuken
+	class ThorCLI < Thor
+
+		class Install < Thor::Group
+			include Thor::Actions
+
+			class_option "path",
+        aliases: ["-p"],
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.